lib/player.rb

```diff
@@ -1,10 +1,11 @@
 class Player
-    attr_reader :board, :cruiser, :submarine, :ships
+    attr_reader :board, :cruiser, :submarine, :ships, :ships_sunk
     def initialize
         @board = Board.new
         @cruiser = Ship.new("Cruiser", 3)
         @submarine = Ship.new("Submarine", 2)
         @ships = [@cruiser, @submarine]
+        @ships_sunk = 0
     end
 
```

Comment:
Not sure how much bigger this file is going to get since we're keeping a lot of it in turn.rb - I like the counter for ships sunk in each class.